### PR TITLE
some more instruction for lecture 2 on git-based collaboration

### PR DESCRIPTION
**Pull request by SanliFaez**
the assignment itself will be developed elsewhere in the coming weeks

### CODE REVIEW

**Comment by erikvansebille** [comment]
Nice slides for lecture 1, Sanli! Which programme/software did you make these in? Do you want me to append? Or shall I just create and upload my own slide deck?

**Comment by SanliFaez** [comment]
@erikvansebille I am using slides.com (trying to reduce dependence on MS) which give HTML and MD compatible formats. I suggest that you first share your separate slide deck so that we can compare which story can land better for the majority of students. what do you say? 
we can ask TAs to judge the storylines.

File: CourseDevelopment/Week2/Lecture2.md [hunk 1]
```diff
@@ -1,75 +1,45 @@
...
 
 2. **Project management**: 
     - Project management and using issues/labels, introduction to Getting Things Done (GTD) boards. 
-    - Some tips and suggestions effective collaboration and project managment on git platforms
-
-## Tutorial: Week 2
-**Topics in tutorial:**
-- Between the two lectures, students will be given a complex coding task to complete in two teams, starting from forking an incomplete repository and then making the complete project work as a group! 
-- To make sure that every student practices the basic git commands, every student must be able to run the final program on their PC
+    - Some tips and suggestions effective collaboration on git platforms and the essentials of community management.
```
Comment (erikvansebille) [inline CourseDevelopment/Week2/Lecture2.md:26]:
```suggestion
    - Some tips and suggestions on effective collaboration using git platforms, and the essentials of community management.
```

File: CourseDevelopment/Week4/Lecture4.md [hunk 2]
```diff
@@ -1,142 +1,109 @@
...
-
+![Project documentation illustrated as a roadmap](reproducibility.jpg)
+[The Turing Way project](https://the-turing-way.netlify.app/reproducible-research/reproducible-research) illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: 10.5281/zenodo.3332807.
+
+### Lecture format
+During weeks 4 and 5 of the course, students form teams to make a small computer-assisted measurement project, with a focus on reproducibility of the results. Week 4 is mainly for setting up the project and its documentation, and week 5 is dedicated to reproducing the project by other teams and writing a peer-feedback report. 
+
+Students will also get a tour or [Lili's proto lab](https://www.uu.nl/lpl) and get familiar with various rapid-prototyping tool, which they can use to enhance their projects.
+
+**The 5-minutes exercise:** Chose a previous report (at least 6 months old) you have written (e.g. your bachelor project) and pick a random graph that you have generated. In 5 minutes, find the data corresponding to that graph and regenerate it. What was the document structure and was it useful? Can someone else use your documents to reproduce your results?
```
Comment (erikvansebille) [inline CourseDevelopment/Week4/Lecture4.md:20]:
```suggestion
**The 5-minutes exercise:** Choose a previous report (at least 6 months old) you have written (e.g. your bachelor project) and pick a random graph or figure that you have generated. In 5 minutes, find the data corresponding to that graph and regenerate it. What was the documentation structure and was it useful? Can someone else use your documentation to reproduce your results?
```

File: CourseDevelopment/Week4/OpenHardwareExamples.md
Comment (erikvansebille) [inline CourseDevelopment/Week4/OpenHardwareExamples.md:1]: Does this file need a title/heading?

File: CourseDevelopment/Week5/Lecture5.md [hunk 3]
```diff
@@ -1,139 +1,37 @@
...
-+ Week 3(?) https://waag.org/nl/article/hollandse-luchten-sensorkit/ 
-
-Open hardware makers
-+ https://twitter.com/openHWmakers 
-+ https://www.youtube.com/@openHWmakers 
-+ https://openhardware.space/ 
 
 
-## Information Sources / Bibliography
+## Information Sources / Bibliography (WIP: needs a clean-up )
```
Comment (erikvansebille) [inline CourseDevelopment/Week5/Lecture5.md:34]: Still to be done?

File: Projects/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+## About the ./projects directory
+
+To keep the repository tidy, files for each subproject MUST be kept inside a subdirectory in ./projects/
+
+To create the subdirectory, use the naming convention ProjectName_TeamMembers.
+
+Create the necessary directory structure inside the project subdirector, and write the navigation instructions in a README.md file inside that subfolder.
+The preferred text style is Markdown.
```
Comment (erikvansebille) [inline Projects/README.md:8]: Add a link to a Markdown tutorial?

File: .idea/workspace.xml
Comment (erikvansebille) [inline .idea/workspace.xml:1]: This file can/should be deleted?

File: CourseDevelopment/Week5/Lecture5.md (hunk 3 above)
Comment (SanliFaez) [reply CourseDevelopment/Week5/Lecture5.md:34]: yes, soon!

File: .idea/workspace.xml
Comment (SanliFaez) [reply .idea/workspace.xml:1]: yes, they are generated by pycharm. I try to add them to gitignore, but they keep showing up. don't know a solution.

File: Projects/README.md (hunk 4 above)
Comment (SanliFaez) [reply Projects/README.md:8]: will do in the next PR

**Comment by SanliFaez** [comment]
I am merging this PR so that the changes are propagating to the other newer branches.

**Pull request merged**